### PR TITLE
audio: log first few bytes only of tag value changes.

### PR DESCRIPTION
**Pull request by kingosticks**
These really clutter the debug log output and I am yet to find a case where it was interesting. Couldn't think of a better name for this function, open to suggestions!

### CODE REVIEW

**Comment by kingosticks** [comment]
Github is right, I should have written a test.

**Comment by kingosticks** [comment]
@jodal is `repr_tags` a sensible name?

File: mopidy/audio/tags.py [hunk 1]
```diff
@@ -10,6 +10,26 @@
...
+    :type taglist: :class:`Gst.TagList`
+    :param max_bytes: The maximum number of bytes to show for bytes tag values.
+    :type max_bytes: int
+    :rtype: string
+    """
+    result = dict(taglist)
+    for tag_values in result.values():
+        for i, val in enumerate(tag_values):
+            if type(val) is bytes:
+                tag_values[i] = val[:max_bytes]
```
Comment (jodal) [inline mopidy/audio/tags.py:29]: Maybe add a '...' suffix to indicate that there's more here?

**Review by jodal — APPROVED** [review]
LGTM. I think the name is good enough. Not exactly public API either.

**Pull request merged**